bazel/swift_header_collector.bzl

```diff
@@ -0,0 +1,23 @@
+"""
+Propagate the generated Swift header from a swift_library target
+
+This exists to work around https://github.com/bazelbuild/rules_swift/issues/291
+"""
+
+def _swift_header_collector(ctx):
+    dep = ctx.attr.deps[0]
+    return [
+        DefaultInfo(
+            files = dep[CcInfo].compilation_context.headers,
```

Comment:
I'm surprised it's not a `CcInfo` or `Objc`

Reply:
Surprised what isn't? What I'm propagating? It's just a file list used by `hdrs`

Reply:
ah ok